### PR TITLE
Release/3.6.4

### PR DESCRIPTION
**Pull request by mgoldman-globality**
adding AsciiEncodedStringField

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -21,6 +21,7 @@
     python_requires=">=3.6",
     keywords="microcosm",
     install_requires=[
+        "Flask<2",
```
Comment (sammy-globality) [inline setup.py:24]: What breaks with Flask>=2?

Comment (adeleinr) [reply setup.py:24]: I have the same question

**Review by adeleinr — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (adeleinr) [reply setup.py:24]: Sorry that question was missing context, what breaks if you dont use Frask <2 is the tests for the PR Tony just merged before this one. I talked to him and he checked his tests and thinks that flask >=2 is whats breaking them so he suggested to downgrade flast. But, I am not sure exactly how it is breaking those tests, if it is worth noting it at least in the PR.

Comment (mgoldman-globality) [reply setup.py:24]: I'll look into it and see if I can figure out why it's breaking them

Comment (adeleinr) [reply setup.py:24]: Thanks, lets not block this PR on that, but if you do find out let us know

**Review by sammy-globality — APPROVED** [review]

**Pull request merged**